.prospector.yaml

```diff
@@ -0,0 +1,27 @@
+strictness: medium     # default strictness
+test-warnings: false   # inspect unittests
+doc-warnings: true     # inspect docstrings
+
+ignore-paths:          # ignore the docs folder
+  - docs
+  - ci_scripts
+  - examples
+
+pep8:
+  disable:
+    - N806              # ignore pep8 naming (func arg should be lowercase i.e. X_pred)
+  options:
+    max-line-length: 120
+
+pylint:
+  disable:
+    - C0326              # ignore pylint bad-whitespace  (func arguments)
```

Comment:
Is there a reason to not check this one?

Reply:
We have a bunch of argument in functions like https://github.com/automl/SMAC3/blob/5775eaf234730c8aaaa06e46d0eb6e0a8348929e/smac/epm/rf_with_instances.py#L117
which all would create issues.

Reply:
So either we replace all these function arguments that have uppercase letters with lower case ones or we ignore this one rule

Reply:
I actually meant `C0326              # ignore pylint bad-whitespace  (func arguments)`.

Reply:
Oh sorry my bad. I confused myself with the func arguments.
This should be removed. I accidentally left when testing if I configure prospector correctly.